lib/pbench/test/unit/server/database/test_datasets_db.py

```diff
@@ -91,11 +93,6 @@ def test_dataset_metadata_log(self, db_session, create_user, provide_metadata):
             },
         }
 
-    def test_construct_bad_owner(self, db_session):
-        """Test with a non-existent username"""
-        with pytest.raises(DatasetBadParameterType):
-            Dataset(owner="notme", name="fio")
-
```

Comment:
If we decide to keep the relationship between the Dataset table and the User table, I think we'll want this test back.